javascript-bank/account.js

```diff
@@ -0,0 +1,43 @@
+/* exported Account */
+
+function Account(number, holder) {
+  this.number = number;
+  this.holder = holder;
+  this.transactions = [];
+}
+
+Account.prototype.deposit = function (amount) {
+  if (amount > 0 && Number.isInteger(amount) === true) {
+    var output = new Transaction();
+    output.type = 'deposit';
+    output.amount = amount;
```

Comment:
You shouldn't be doing this manually, you should be passing these values as arguments to `Transaction` so it can assign the properties for you